test/test.js

```diff
@@ -691,6 +694,22 @@ describe('Test Schema Repository Tests', function() {
         tests.all(options);
     });
 
+    it('Should fail structure test if schema $id does not match title or path', async function() {
+        const structureTests = rewire('../lib/tests/structure');
+        const assertIdMatchesDirectory = structureTests.__get__('assertIdMatchesDirectory');
+        const options = readConfig({
+            schemaBasePath: fixture.resolve('schemas/'),
+            contentTypes: ['yaml']
+        }, true);
+
+        const { incorrect: incorrectSchemas } = findSchemasByTitle(options);
+
+        assert.throws(
+            () => assertIdMatchesDirectory(incorrectSchemas[0], options),
```

Comment:
nit: could you add some context on this test?

Here you are checking that `$id` matches the path. Shouldn't you also test for a match on the `title` field? Is it done implicitly in `findSchemasByTitle` ?

Reply:
This is done in [another assertion](https://github.com/wikimedia/jsonschema-tools/pull/46/files#diff-0895d97e96519bf5892adf6389b0bb267f03b9d3ebb863cf2d405bd8aaed98d9R188-R190) in the structure test that should probably be extracted out and tested seperately

Reply:
thanks for clarifying @tchin25 . No need to refactor tests IMHO, but would you mind adding a comment or changing the test description so that it matches actual behavior?

Reply:
Done